api-client/version.sbt

```diff
@@ -1 +1 @@
-version := "1.3-SNAPSHOT"
\ No newline at end of file
+version := "1.4"
```

Comment:
not yet :) that's automatically updated when you release so I'd recommend you leave it the way it was